yt/sample_data_registry.json

```diff
@@ -259,6 +259,15 @@
     "load_name": "plt00015",
     "url": "https://yt-project.org/data/Laser.tar.gz"
   },
+  "MagneticumCluster.tar.gz": {
```

Comment:
in order for this to integrate with yt.load_sample, you'll need to update https://github.com/yt-project/website/blob/master/data/datafiles.json

Reply:
@neutrinoceros This was done already (you looked at the PR): https://github.com/yt-project/website/pull/96

Reply:
I didn't realise it was the same one. In any case I've tested this branch and it currently doesn't work
```python
import yt
yt.load_sample("MagneticumCluster")
```

Reply:
Do you have a traceback or something for more information?